qa/L0_batch_input/test.sh

```diff
@@ -81,13 +81,14 @@ for BACKEND in $BACKENDS; do
     # Use nobatch model to showcase ragged input, identity model to verify
     # batch input is generated properly
     cp -r $IDENTITY_DATADIR/${BACKEND}_nobatch_zero_1_float32 models/ragged_io
+    IO_SUFFIX=$([[ "$BACKEND" == "libtorch" ]] && echo "__0" || echo "0")
```

Comment:
Torch backend shouldn't need this handling now?

Reply:
Oh, you're right. It looks like it was just having issues because the way the model config is generated now, it still uses __0 for input and output. I'll see if I can change the model generation and make sure any tests using it still work fine (if it's too complex, can just update this test).

Reply:
> > > RuntimeError: Expected all tensors to be on the same device, but found at least two devices, cpu and cuda:0! (when checking argument for argument mat2 in method wrapper_CUDA_mm)
> > 
> > 
> > The batch input implementation (on other backends) generates the tensor on CPU, which can be why the mismatch happens, perhaps needs to port the tensor to the same device as regular input after generation.
> 
> I see, thanks. The PR generates it on CPU as well, so can look at how to port it here as well.

Do you know why the other backends don't generate it on GPU in the first place? I tried modifying it to generate on the same device as other inputs [here](https://github.com/triton-inference-server/pytorch_backend/commit/0b291e40c7118602bafebaf3db7793c329de0618#diff-859dcc1d94f2d13b0d19e7a14af430cd18eb66395a6f69f2d4886854319c4467R1880) and the result was a crash during model execution (`void c10::function_ref<void (char**, long const*, long, long)>::callback_fn<at::native::AVX2::VectorizedLoop2d<at::native::(anonymous namespace)::div_true_kernel(at::TensorIteratorBase&)::{lambda()#1}::operator()() const::{lambda()#2}::operator()() const::{lambda(float, float)#1}, at::native::(anonymous namespace)::div_true_kernel(at::TensorIteratorBase&)::{lambda()#1}::operator()() const::{lambda()#1}::operator()() const::{lambda(at::vec::AVX2::Vectorized<float>, at::vec)#2}> >(long, char**, long const*, long, long)`).

Reply:
Generating batch input on CPU because it requires writing to the result buffer one element by element and thus the generation is always on CPU and assuming the framework can port the tensor internally makes things easiler. 

You need to make the change here as well: https://github.com/triton-inference-server/pytorch_backend/commit/0b291e40c7118602bafebaf3db7793c329de0618#diff-859dcc1d94f2d13b0d19e7a14af430cd18eb66395a6f69f2d4886854319c4467R1886

Reply:
Thanks, Guan! That resolved the issue. Debugging wrong output (and output shapes) now, but the model inference is running.